app/src/main/res/drawable/shape_gray_circle.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="oval">
+
+    <solid android:color="#14080434" />
+
+    <size
+        android:width="40dp"
+        android:height="40dp" />
+</shape>
```

Comment:
We already have `ic_circle.xml` in resources, but I was wondering should we replace it with oval `android:shape` or keep it as vector XML. Your call, just keep one and merge them.

Reply:
This shape is imageView's background, tint cant be applied to background (only to android:src)